nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -285,6 +285,10 @@
 
 - Package `cloud-sql-proxy` was renamed to `google-cloud-sql-proxy` as it cannot be used with other cloud providers.;
 
+- The Yama LSM is now enabled by default in the kernel, which prevents ptracing non-child processes.
+  This means you will not be able to attach gdb to an existing process, but will need to start that process from gdb (so it is a child).
+  Or you can set `boot.kernel.sysctl."kernel.yama.ptrace_scope"` to 0.
+
```

Comment:
The 23.11 branch off already happened, when this PR was merged.
Either this PR should be backported or this Note should be moved to the 24.05 Changelog.

Reply:
Is there anything I can do about this?

Reply:
It's not in 23.11, right? So I just have to move the changelog entry in a new PR to `master`.

Reply:
True, indeed!